File: 본문/python/Code_2_04_3.py
# 입력
N, S = map(int, input().split())

# 답 구하기
answer = 0
for i in range(1, N + 1):
	for j in range(1, N + 1):
		if i + j <= S:
			answer += 1

# 출력
print(answer)

File: 본문/cpp/Code_2_04_3.cpp
#include <iostream>
using namespace std;

int N, S;
long long Answer = 0;

int main() {
	// 입력
	cin >> N >> S;

	// 답 구하기
	for (int i = 1; i <= N; i++) {
		for (int j = 1; j <= N; j++) {
			if (i + j <= S) Answer += 1;
		}
	}

	// 출력
	cout << Answer << endl;
	return 0;
}